<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Instrument</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="10,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="txtClassCode"/>
        </item>
        <item row="10" column="2" colspan="2">
         <widget class="QTextEdit" name="txtDetectorDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="txtHardwareVersion"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Class Code:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtClassCode</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Instrument ID:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtInstrumentId</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="txtManufacturer"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Manufacturer:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtManufacturer</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="txtDetector"/>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="txtInstrumentId"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Model Name (required):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtDetector</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Hardware Vers:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtHardwareVersion</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Instrument Description:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QComboBox" name="cmbReplay"/>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="btnNewReplay">
          <property name="text">
           <string>Add new...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Replay Tool:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtHardwareVersion</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="2">
         <widget class="QPushButton" name="manageReplays">
          <property name="text">
           <string>Manage Replay Tools</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Influences</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tblInfluences">
          <column>
           <property name="text">
            <string>Influence</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>0 (offset)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>1 (gain)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>2 (quadratic)</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnImportInfluences">
          <property name="text">
           <string>Import Influences...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnAddBaseSpectra">
       <property name="text">
        <string>Add Base Spectra...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRemoveBaseSpectra">
       <property name="text">
        <string>Remove Base Spectra...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Materials (Base Spectra)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Double-click on a material name to view spectrum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lstBaseSpectra"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Energy Cal (set by base spectra)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>3 (cubic)</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal3</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtEcal0">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>1 (gain)</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal1</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="txtChannelCount">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>chan count:</string>
          </property>
          <property name="buddy">
           <cstring>txtChannelCount</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="txtEcal3">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txtEcal1">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>0 (offset)</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal0</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="txtEcal2">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>2 (quad)</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal2</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QGroupBox" name="includeSecondarySpectrumCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Include secondary spectrum in sample spectra</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="secondaryIsBackgroundRadio">
          <property name="text">
           <string>as background spectrum</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="secondaryIsCalibrationRadio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>as internal calibration spectrum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bckgndSpecFileDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnExportDetector">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export Instrument...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImportDetector">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import Instrument...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtDetector</tabstop>
  <tabstop>txtManufacturer</tabstop>
  <tabstop>txtInstrumentId</tabstop>
  <tabstop>txtClassCode</tabstop>
  <tabstop>txtHardwareVersion</tabstop>
  <tabstop>txtDetectorDescription</tabstop>
  <tabstop>txtEcal0</tabstop>
  <tabstop>txtEcal1</tabstop>
  <tabstop>txtEcal2</tabstop>
  <tabstop>txtEcal3</tabstop>
  <tabstop>txtChannelCount</tabstop>
  <tabstop>lstBaseSpectra</tabstop>
  <tabstop>tblInfluences</tabstop>
  <tabstop>btnImportDetector</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
